include/Utilities/Math.hpp

```diff
@@ -69,8 +72,10 @@ template <typename T> ACTS_DEVICE_FUNC T determinant(T *m, int size) {
   return det;
 }
 
-template <typename T>
-ACTS_DEVICE_FUNC void invert(const ActsMatrixX<T> *em, ActsMatrixX<T> *result) {
+// T is the data type of the computations (default is double)
+// P is the data type of the input/output matrices (aka ActsScalar)
+template <typename T, typename P>
```

Comment:
To make the default to be `double`, shall we write as `template <typename P, typename T = double>`?

Reply:
I defined it like this: 
`// the precision of the matrix inversion computations`
`using T = double;`
But I have nothing against your way of writing it. I will make the change now.